doc/api/crypto.md

```diff
@@ -6142,6 +6142,7 @@ See the [list of SSL OP Flags][] for details.
 [`hash.update()`]: #hashupdatedata-inputencoding
 [`hmac.digest()`]: #hmacdigestencoding
 [`hmac.update()`]: #hmacupdatedata-inputencoding
+[`import()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports
```

Comment:
Just a little thing, which you can definitely ignore: could you use a link which has no locale in it? This way people might get a content in their native language.

Reply:
In general, I'd love to do that. However, it seems like some of the translations on MDN are incomplete and don't mention `import()` at all (for example, the German translation) and the other translations have localized the anchor `#dynamic_imports` so that the link does not jump to the right section (for example, the French translation). I am not sure if it makes more sense to link from the English documentation to the English MDN page that we know how to link to.

Reply:
I see. Don't worry then, better have a localized non-broken link then!